app/scripts/modules/core/loadBalancer/loadBalancer.states.ts

```diff
@@ -73,9 +73,7 @@ module(LOAD_BALANCER_STATES, [
       name: 'loadBalancers',
       views: {
         'nav': {
-          templateUrl: require('../loadBalancer/filter/filterNav.html'),
-          controller: 'LoadBalancerFilterCtrl',
-          controllerAs: 'loadBalancerFilters'
+          template: '<load-balancer-filter app="$resolve.app"></load-balancer-filter>',
```

Comment:
Note the addition of route-to-component-template and utilizing the `$resolve` scope variable.